csharp/ql/lib/semmlecode.csharp.dbscheme

```diff
@@ -199,7 +199,7 @@ sourceLocationPrefix(
               | @destructor | @callable_accessor | @value_or_ref_type | @declaration_with_accessors
               | @local_function | @lambda_expr;
 
-/** LOCATIONS, ASEMMBLIES, MODULES, FILES and FOLDERS **/
+/** LOCATIONS, ASSEMBLIES, MODULES, FILES and FOLDERS **/
```

Comment:
I suggest to revert this change, as we would otherwise need to create upgrade/downgrade scripts.